css/index.css

```diff
@@ -66,6 +160,15 @@ p {
     line-height: 1.4;
 }
 
+.top-nav>* {
```

Comment:
No need to use `>*` here. You can use `nav` to select the `nav` tag. By using `>*` you're applying your flexbox rules to the child elements of `nav`, which isn't what we want.